src/DotNetLightning.Core/Channel/Channel.fs

```diff
@@ -676,104 +701,108 @@ module Channel =
-            { c with State = WaitForFundingLocked { prevState with OurMessage = msg; HaveWeSentFundingLocked = true } }
+            { channel with State = WaitForFundingLocked { prevState with OurMessage = msg; HaveWeSentFundingLocked = true } }
         | BothFundingLocked data, WaitForFundingSigned _s ->
-            { c with State = ChannelState.Normal data }
+            { channel with State = ChannelState.Normal data }
         | BothFundingLocked data, WaitForFundingLocked _s ->
-            { c with State = ChannelState.Normal data }
+            { channel with State = ChannelState.Normal data }
 
         // ----- normal operation --------
-        | WeAcceptedCMDAddHTLC(_, newCommitments), ChannelState.Normal d ->
-            { c with State = ChannelState.Normal({ d with Commitments = newCommitments }) }
-        | WeAcceptedUpdateAddHTLC(newCommitments), ChannelState.Normal d ->
-            { c with State = ChannelState.Normal({ d with Commitments = newCommitments }) }
-
-        | WeAcceptedCMDFulfillHTLC(_, newCommitments), ChannelState.Normal d ->
-            { c with State = ChannelState.Normal({ d with Commitments = newCommitments }) }
-        | WeAcceptedFulfillHTLC(_msg, _origin, _htlc, newCommitments), ChannelState.Normal d ->
-            { c with State = ChannelState.Normal({ d with Commitments = newCommitments }) }
-
-        | WeAcceptedCMDFailHTLC(_msg, newCommitments), ChannelState.Normal d ->
-            { c with State = ChannelState.Normal({ d with Commitments = newCommitments }) }
-        | WeAcceptedFailHTLC(_origin, _msg, newCommitments), ChannelState.Normal d ->
-            { c with State = ChannelState.Normal({ d with Commitments = newCommitments }) }
-
-        | WeAcceptedCMDFailMalformedHTLC(_msg, newCommitments), ChannelState.Normal d ->
-            { c with State = ChannelState.Normal({ d with Commitments = newCommitments }) }
-        | WeAcceptedFailMalformedHTLC(_origin, _msg, newCommitments), ChannelState.Normal d ->
-            { c with State = ChannelState.Normal({ d with Commitments = newCommitments }) }
-
-        | WeAcceptedCMDUpdateFee(_msg, newCommitments), ChannelState.Normal d ->
-            { c with State = ChannelState.Normal({ d with Commitments = newCommitments }) }
-        | WeAcceptedUpdateFee(_msg), ChannelState.Normal _d -> c
-
-        | WeAcceptedCMDSign(_msg, newCommitments), ChannelState.Normal d ->
-            { c with State = ChannelState.Normal({ d with Commitments = newCommitments }) }
-        | WeAcceptedCommitmentSigned(_msg, newCommitments), ChannelState.Normal d ->
-            { c with State = ChannelState.Normal({ d with Commitments = newCommitments }) }
-
-        | WeAcceptedRevokeAndACK(newCommitments), ChannelState.Normal d ->
-            { c with State = ChannelState.Normal({ d with Commitments = newCommitments }) }
+        | WeAcceptedCMDMonoHopUnidirectionalPayment(_, newCommitments), ChannelState.Normal stateData ->
```

Comment:
@canndrew I only meant for renaming of variables on new code, not on DNL code that is already there, we want our delta between Joe's DNL as minimal as possible

Reply:
Ah. I can change it back except for the renaming of `d => stateData` then.

Reply:
yes please